packages/expo-video/android/src/main/java/expo/modules/video/VideoPlayer.kt

```diff
@@ -154,6 +159,8 @@ class VideoPlayer(context: Context, appContext: AppContext, source: VideoSource?
         setStatus(playerStateToPlayerStatus(player.playbackState), null)
       }
 
+      this@VideoPlayer.duration = 0f
+
```

Comment:
We should move this inside of 
```
 error?.let {
        setStatus(ERROR, error)
      }
```
When it's here it will reset to 0 even when error is set to nil :)